src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -205,8 +223,12 @@ namespace OpenMS
     static constexpr const char* INTEGRATION_TYPE_SIMPSON = "simpson";
     /// Baseline type: base to base
     static constexpr const char* BASELINE_TYPE_BASETOBASE = "base_to_base";
-    /// Baseline type: vertical division
+    /// Baseline type: vertical division (min of end points; only for backwards compatibility)
```

Comment:
For naming, I would just suggest using "vertical_division" and "vertical_division_max" and not using "vertical_division_min".  The name "vertical_division" is pretty ubiquitous in most peak integration software (at least for small molecules).

Reply:
I am ok with both, its just that using min/max would be less confusing for people.

Reply:
That works me.  Ready to be merged?